### PR TITLE
feat: Auto clean transaction log

### LINKED ISSUE #556: 【XA】build auto clean transaction log ability
**What would you like to be added**:

**Why is this needed**:

### PR DESCRIPTION
**Pull request by PangXing**
**What this PR does**:
 Tx clean log
**Which issue(s) this PR fixes**:

Fixes #556 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by dongzl — APPROVED** [review]

File: pkg/runtime/transaction/trx_log.go [hunk 1]
```diff
@@ -68,16 +69,19 @@ CREATE TABLE IF NOT EXISTS __arana_trx_log
 // TxLogManager Transaction log management
 type TxLogManager struct {
 	sysDB proto.DB
+	timer *time.Timer
 }
 
 // init executes create __arana_tx_log table action
-func (gm *TxLogManager) init() error {
+func (gm *TxLogManager) init(delay time.Duration) error {
```
Comment (Mulavar) [inline pkg/runtime/transaction/trx_log.go:76]: when should we call this method for init? I did not find any caller of the method.

File: pkg/runtime/transaction/trx_log.go [hunk 2]
```diff
@@ -179,5 +183,26 @@ func (gm *TxLogManager) ScanTxLog(pageNo, pageSize uint64, conditions []Conditio
 // partition table according to the day level or hour level.
 // the execution of this task requires distributed task preemption based on the metadata DB
 func (gm *TxLogManager) runCleanTxLogTask() {
-
+	var pageNo uint64 = 0
```
[the remark this answers is not in the document]
Comment (PangXing) [reply pkg/runtime/transaction/trx_log.go]: Done

File: pkg/runtime/transaction/trx_log.go [hunk 3]
```diff
@@ -179,5 +183,26 @@ func (gm *TxLogManager) ScanTxLog(pageNo, pageSize uint64, conditions []Conditio
...
+	var pageSize uint64 = 50
+	cond := Condition{
+		FiledName: "status",
+		Operation: Equal,
+		Value:     runtime.TrxFinish,
+	}
```
[the remark this answers is not in the document]
Comment (PangXing) [reply pkg/runtime/transaction/trx_log.go]: Done

File: pkg/runtime/transaction/trx_log.go (hunk 1 above)
Comment (PangXing) [reply pkg/runtime/transaction/trx_log.go:76]: Inner function "init" has been changed to Outer function "Init". It should be Init In function of CreateTrxManager at once.

**Review by dongzl — APPROVED** [review]

**Review by Lvnszn — APPROVED** [review]

File: pkg/runtime/transaction/trx_manager.go [hunk 4]
```diff
@@ -28,6 +29,7 @@ import (
 
 var (
 	ErrorTrxManagerNotInitialize = errors.New("TrxManager not initialize")
+	DefaultCleanLogDelay         = 1 * time.Hour
```
Comment (Lvnszn) [inline pkg/runtime/transaction/trx_manager.go:32]: Looks like you don't need 1 *

**Review by cjphaha — APPROVED** [review]

**Pull request merged**